.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor/
 composer.lock
+/log/
```

Comment:
Can you also add the Circle CI implementation? Check your email to see what he is looking for.